CMakeLists.txt

```diff
@@ -489,16 +503,14 @@ if(MSVC)
   endforeach(flag_var)
 
   # Try harder
-  list(APPEND CUDA_NVCC_FLAGS "-Xcompiler" "/w" "-w")
+  string(APPEND CMAKE_CUDA_FLAGS " -Xcompiler /w -w")
```

Comment:
Hmm, can you please explain why do you want to go from `list` to `string`? `string(APPEND` is inherently bad if code is compiled in folder than have spaces in it.

Reply:
A list is a string separated with semicolons, but `CMAKE_CUDA_FLAGS` expects a normal string so it will pass the semicolons into the command line invocation.